BookingsApi/BookingsApi.Common/Services/FeatureToggles.cs

```diff
@@ -1,41 +0,0 @@
-﻿using System;
-using LaunchDarkly.Logging;
-using LaunchDarkly.Sdk;
-using LaunchDarkly.Sdk.Server;
-using LaunchDarkly.Sdk.Server.Interfaces;
-
-namespace BookingsApi.Common.Services
-{
-    public interface IFeatureToggles
-    {
-        bool UsePostMay2023Template();
-    }
-
-    public class FeatureToggles : IFeatureToggles
-    {
-        private readonly ILdClient _ldClient;
-        private readonly Context _context;
-        private const string LdUser = "vh-booking-api";
-        private const string NewNotifyTemplatesToggleKey = "notify-post-may-2023-templates";
-
-        public FeatureToggles(string sdkKey, string environmentName)
-        {
-            var config = LaunchDarkly.Sdk.Server.Configuration.Builder(sdkKey)
-                .Logging(Components.Logging(Logs.ToWriter(Console.Out)).Level(LogLevel.Warn)).Build();
-            _context = Context.Builder(LdUser).Name(environmentName).Build();
-            _ldClient = new LdClient(config);
-        }
-        
-        public bool UsePostMay2023Template() => GetBoolToggle(NewNotifyTemplatesToggleKey);
-        
-        private bool GetBoolToggle(string key)
-        {
-            if (!_ldClient.Initialized)
-            {
-                throw new InvalidOperationException("LaunchDarkly client not initialized");
-            }
-            return _ldClient.BoolVariation(key, _context);
-        }
-    }
-}
- 
```

Comment:
I dont think we need to delete the whole implementation of the feature toggles class. I know that it's no feature toggles in use now, but just removing the individual toggle would suffice. Otherwise we need to reimplement this when we have a new feature flag to add